packages/xml-views-validation/src/validators/elements/use-of-deprecated-class.ts

```diff
@@ -13,40 +14,52 @@ export function validateUseOfDeprecatedClass(
     return [];
   }
 
-  if (ui5Class.deprecatedInfo !== undefined) {
+  if (
+    ui5Class.deprecatedInfo !== undefined &&
+    // An issue without a position is not a useful issue...
+    (xmlElement.syntax.openName !== undefined ||
+      /* istanbul ignore next - defensive programing for (currently) none reproducible scenario */
```

Comment:
Does this ignore comment work in the middle of the if condition?

Reply:
Yes I had coverage error on specific IF condition line and had to use this to fix it.